File: backend-dotnet/Application/Commands/Comments/CreateCommentCommand.cs
using Application.DTOs;

namespace Application.Commands.Comments
{
    public class CreateCommentCommand : ICommand<CommentResponseDto>
    {
        public Guid PostId { get; set; }
        public Guid UserId { get; set; }
        public CreateCommentDto Dto { get; set; } = null!;
    }
}

File: backend-java/src/main/java/com/jerrygram/application/commands/comments/CreateCommentCommand.java
package com.jerrygram.application.commands.comments;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CreateCommentCommand {
    private UUID postId;
    private UUID authorId;
    private String content;
}